examples/object-initializer/object_initializer.out

```diff
@@ -2,4 +2,4 @@
 # `.bal` file, and execute the `ballerina run` command.
 $ ballerina run object_initializer.bal
 {name:"John"}
-Invalid name format {}
+error The age should be between 0-100
```

Comment:
```suggestion
Error: The age should be between 0-100.
```

Reply:
Ballerina currently prints error messages as above, since this is a .out file shall we keep it as is?